### ISSUE TITLE
Result pane background not saved

### ISSUE THREAD

**Issue by nblackburn** [parsify-dev/desktop#21]
When changing the result pane backgroud, the value doesn't appear to be saved.

**Comment by nblackburn** [comment]
Related #9

**Comment by xxczaki** [comment]
I wasn't able to reproduce this issue. It can be related to configuration storage and might occur when auto-updating from certain older versions. You can either try reinstalling the app or manually changing the config file (instructions below).

---

The configuration file is stored as `config.json` inside `Parsify Desktop` folder in the following directory:

- when on macOS, `~/Library/Application Support`;
- when on Windows, `%APPDATA%`;
- when on Linux, `$XDG_CONFIG_HOME` or `~/.config`

If you don't care about saved preferences, go ahead and delete the file (this will force the app to regenerate it). However, if you want to save your current preferences, open it in a text editor and add the following to `preferences`:

```diff
	"preferences": [
		{
			"name": "precision",
			"value": 4
		},
		{
			"name": "dateFormat",
			"value": "MM/dd/yyyy HH:mm"
		},
		{
			"name": "alwaysOnTop",
			"value": false
		},
		{
			"name": "border",
			"value": true
		},
		{
			"name": "background",
			"value": ""
		},
		{
			"name": "text",
			"value": "#fff"
		},
		{
			"name": "result",
			"value": "#8dd32d"
		},
		{
			"name": "resultBg",
			"value": ""
		},
		{
			"name": "borderColor",
			"value": "#5f616b"
		},
		{
			"name": "syntaxHighlighting",
			"value": true
		},
		{
			"name": "unitValidation",
			"value": true
		},
		{
			"name": "configuration",
			"value": ""
		},
		{
			"name": "alwaysOnTop",
			"value": false
		},
		{
			"name": "dateFormat",
			"value": "MM/dd/yyyy HH:mm"
+		},
+		{
+			"name": "resultBg",
+			"value": "#27282d"
+		}
	]
```

**Comment by nblackburn** [comment]
![2020-06-11_17-34-01](https://user-images.githubusercontent.com/2931085/84415005-10385880-ac0a-11ea-8767-d3c15ef36c7a.gif)


```json
{
	"preferences": [
		{
			"name": "precision",
			"value": 4
		},
		{
			"name": "dateFormat",
			"value": "MM/dd/yyyy HH:mm"
		},
		{
			"name": "alwaysOnTop",
			"value": false
		},
		{
			"name": "border",
			"value": false
		},
		{
			"name": "background",
			"value": "#222222"
		},
		{
			"name": "text",
			"value": "#ffffff"
		},
		{
			"name": "result",
			"value": "#ffffff"
		},
		{
			"name": "resultBg",
			"value": "#27282d"
		},
		{
			"name": "borderColor",
			"value": "#444444"
		},
		{
			"name": "syntaxHighlighting",
			"value": true
		},
		{
			"name": "unitValidation",
			"value": true
		},
		{
			"name": "configuration",
			"value": ""
		},
		{
			"name": "alwaysOnTop",
			"value": false
		},
		{
			"name": "dateFormat",
			"value": "MM/dd/yyyy HH:mm"
		},
		{
			"name": "resultBg",
			"value": "#27282d"
		}
	],
	"units": [],
	"plugins": [
		{
			"name": "parsify-plugin-covid19",
			"on": true
		}
	],
	"expressions": [
		"ten = 10 usd in gbp",
		"",
		"20% of ten"
	],
	"__internal__": {
		"migrations": {
			"version": "5.2.0"
		}
	}
}
```

**Comment by nblackburn** [comment]
I have also tried removing the application and the folders used by it but this continues to happen.

**Comment by xxczaki** [comment]
@nblackburn Thanks for providing a visual reproduction of this issue. I managed to fix it in version `1.2.0` of Parsify Desktop, which should be available to download in a few hours. It includes some major changes to how user preferences are being stored, so a manual installation will be recommended.

Full changelog will be available here: https://headwayapp.co/parsify-changelog

**Comment by xxczaki** [comment]
Should be fixed in 1.2.0 :)
